docs/source/quicktour.mdx

```diff
@@ -14,186 +14,188 @@ specific language governing permissions and limitations under the License.
+>>> from transformers import AutoTokenizer, TFAutoModelForSequenceClassification
+
 >>> model = TFAutoModelForSequenceClassification.from_pretrained(model_name, from_pt=True)
 >>> tokenizer = AutoTokenizer.from_pretrained(model_name)
->>> classifier = pipeline("sentiment-analysis", model=model, tokenizer=tokenizer)
 ```
 
-If you don't find a model that has been pretrained on some data similar to yours, you will need to fine-tune a
-pretrained model on your data. We provide [example scripts](examples) to do so. Once you're done, don't forget
-to share your fine-tuned model on the hub with the community, using [this tutorial](model_sharing).
+Then you can specify the model and tokenizer in the [`pipeline`], and apply the `classifier` on your target text:
 
-<a id='pretrained-model'></a>
+```py
+>>> classifier = pipeline('sentiment-analysis', model=model, tokenizer=tokenizer)
+>>> classifier("Nous sommes très heureux de vous présenter la bibliothèque 🤗 Transformers.")
+[{"label": "5 stars", "score": 0.7272651791572571}]
+```
 
-## Under the hood: pretrained models
+If you can't find a model for your use-case, you will need to fine-tune a pretrained model on your data. Take a look at our [fine-tuning tutorial](./training) to learn how. Finally, after you've fine-tuned your pretrained model, please consider sharing it (see tutorial [here](./model_sharing)) with the community on the Model Hub to democratize NLP for everyone! 🤗
 
-Let's now see what happens beneath the hood when using those pipelines.
+## AutoClasses
 
 <Youtube id="AhChOFRegn4"/>
 
-As we saw, the model and tokenizer are created using the `from_pretrained` method:
+Under the hood, the [`AutoModel`] and [`AutoTokenizer`] classes work together to power the [`pipeline`]. The [`AutoClasses`](./model_doc/auto) are shortcuts that automatically retrieves the model architecture of a pretrained model from it's name or path. You only need to select the appropriate [`AutoModel`] for your task and it's associated tokenizer with [`AutoTokenizer`]. 
+
+Let's return to our example and see how we can use the `AutoClasses` to replicate the results of the [`pipeline`].
+
+### Using the AutoTokenizer
+
+A tokenizer is responsible for preprocessing text in a way that is understandable by the model. First, the tokenizer will split a given text into words called *tokens*. There are multiple rules that govern the tokenization process, including how to split a word and at what level (learn more [here](./tokenizer_summary)). The most important thing to remember though is you need to instantiate the tokenizer with the name of the model to ensure you're using the same tokenization rules a model was pretrained with.
+
+Load a tokenizer with [`AutoTokenizer`]:
 
 ```py
 >>> from transformers import AutoTokenizer, AutoModelForSequenceClassification
 
->>> model_name = "distilbert-base-uncased-finetuned-sst-2-english"
->>> pt_model = AutoModelForSequenceClassification.from_pretrained(model_name)
```

Comment:
If we don't load the model in this code sample, let's remove the framework switch and the imports.